tests/Hooks/AfterEachTest.php

```diff
@@ -1,18 +1,14 @@
 <?php
 
 uses()->afterEach(function () {
-    expect($this)
-        ->toHaveProperty('ith')
```

Comment:
`toHaveProperty` expectation doesn't work for properties handled with __get method